plugins/woocommerce-admin/client/activity-panel/activity-panel.js

```diff
@@ -164,9 +164,9 @@ export const ActivityPanel = ( { isEmbedded, query } ) => {
 			ONBOARDING_STORE_NAME
 		);
 
-		const setupList = getTaskList( activeSetupList );
+		const setupList = activeSetupList && getTaskList( activeSetupList );
 
-		const isSetupTaskListHidden = setupList?.isHidden ?? true;
+		const isSetupTaskListHidden = setupList?.isHidden ?? false;
```

Comment:
Odd that the way this was written, ie if `setupList` was null or undefined, `isSetupTaskListHidden` would evaluate to `true`. I wasn't able to spot any downstream problems with this logic reversed though 🤷🏽

Reply:
So when `isSetupTaskListHidden` is true, https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/activity-panel/activity-panel.js#L177 requests the API endpoints. Since `isSetupTaskListHidden` always evaulates to true initially, the APIs are requested regardless of the final value of `isSetupTaskListHidden`

Reply:
> This is testing well, thanks @moon0326. However I see /wp-json/wc-analytics/reports/performance-indicators still being called. That endpoint powers these stats and can be also disabled when the TaskList is present.

Good catch 👍 I'll work on a patch.

Reply:
@psealock  I've looked into the [code](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/homescreen/stats-overview/index.js#L141) and it doesn't check for the Tasklist visibility.  Since stores can have orders regardless of the TaskList status, wouldn't it be expected to query the endpoint? 🤔

Reply:
>Since stores can have orders regardless of the TaskList status, wouldn't it be expected to query the endpoint? 🤔

If I remember correctly, the `performance-indicators` endpoint just powers those stats I screenshoted which are hidden when the task list is available. I don't think they are used elsewhere like the Orders page.

Reply:
>Those stats are visible all the time for me for some reason.

In that case we should leave the endpoints in place then. Thanks for double checking